module-bastion.tf

```diff
@@ -33,12 +33,12 @@ module "bastion" {
   assign_dns          = var.assign_dns
   availability_domain = coalesce(var.bastion_availability_domain, lookup(local.ad_numbers_to_names, local.ad_numbers[0]))
   image_id            = local.bastion_image_id
-  nsg_ids             = concat(var.bastion_nsg_ids, var.create_nsgs ? [module.network.bastion_nsg_id] : [])
+  nsg_ids             = compact(flatten([var.bastion_nsg_ids, [module.network.bastion_nsg_id]]))
```

Comment:
`var.create_nsgs` is removed; nsg_id output from network module will be null if not created, and removed from the result via `flatten`/`compact`.